corehq/motech/repeaters/static/repeaters/js/add_form_repeater.js

```diff
@@ -1,4 +1,12 @@
-hqDefine("repeaters/js/add_form_repeater", function() {
+hqDefine("repeaters/js/add_form_repeater", [
+    'jquery',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/widgets',          // case repeaters use .ko-select2
+    'locations/js/widgets_main',    // motech repeaters use the SupplyPointSelectWidget
```

Comment:
what's the distinction between a case repeater and a "motech" repeater? All repeaters are "motech" AFAIK.

Reply:
oops, this should be OpenMRS, not motech